common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1272,6 +1272,12 @@
     ],
     "sqlState" : "42710"
   },
+  "FIELD_INDEX_ON_ROW_WITHOUT_SCHEMA" : {
```

Comment:
Could you create a sub-class of `UNSUPPORTED_CALL` like:
```
UNSUPPORTED_CALL.FIELD_INDEX
```
with simple explanation like `The row shall have a schema to get an index of the field <fieldName>.`

Reply:
Sounds good. Just applied the comment. Thanks!